components/unisonic/lm317a.stanza

```diff
@@ -42,3 +44,6 @@ public pcb-component component :
   property(self.reference-voltage) = min-typ-max(1.20, 1.25, 1.3)
   property(self.minimum-load) = 4.5e-3
   property(self.adj-current) = 50.0e-6
+  property(self.max-current) = 1.5 ; A
+  ; property(self.output.power-pin) = PowerSupplyPin(v-out, 0.5)
```

Comment:
Do these commented out properties need to be in the component model?

Reply:
I left the commented out because I was unsure how to set them correctly. At the component level we don't know the target output voltage. If they become useless, I can delete the properties

Reply:
Let's delete for now - Mike has pointed out commented code has a lot of ambiguities, and can be added back pretty easily

Reply:
Ok, np.